pkg/pipeline/resource_pipeline_project_integration.go

```diff
@@ -92,6 +97,11 @@ func pipelineProjectIntegrationResource() *schema.Resource {
 						Required:    true,
 						Description: "Value of the input property.",
 					},
+					"is_sensitive": {
+						Type:        schema.TypeBool,
+						Optional:    true,
```

Comment:
I don't think defaulting this attribute to `true` is sensible. I think it's less common for attribute to be sensitive.

Reply:
Good idea; updated.